### PR TITLE
Fix caustics no rendering if shadow data is disabled

### PR DESCRIPTION
**Pull request by daleeidd**
When shadow data is disabled the SS shadow texture was not set. This PR binds that texture to a white texture if disabled.

### CODE REVIEW

File: crest/Assets/Crest/Crest/Scripts/LodData/Shadows/LodDataMgrShadow.cs [hunk 1]
```diff
@@ -120,6 +120,11 @@ internal override void OnDisable()
         {
             base.OnDisable();
 
+            {
```
Comment (huwb) [inline crest/Assets/Crest/Crest/Scripts/LodData/Shadows/LodDataMgrShadow.cs:123]: Not certain the additional scope is necessary but that's minor

**Review by huwb — APPROVED** [review]
Looks good

File: crest/Assets/Crest/Crest/Scripts/LodData/Shadows/LodDataMgrShadow.cs (hunk 1 above)
Comment (daleeidd) [reply crest/Assets/Crest/Crest/Scripts/LodData/Shadows/LodDataMgrShadow.cs:123]: It's for adding a built-in renderer conditional downstream. Setting up the scope here reduces conflicts. I'll add a comment.

**Comment by daleeidd** [comment]
Thanks.

**Pull request merged**